portfolio/src/main/webapp/script.js

```diff
@@ -41,15 +41,24 @@ function getComments() {
 
   fetch('/comments').then(response => response.json()).then((cmtContainer) => {
 
-    if(!cmtContainer) return;
+    if (!cmtContainer) return;
 
-    var str_comments = "<h2>Comments</h2>";
+    var commentsOutput = document.getElementById("comments-output");
+
+    var cmtIntroDOM = document.createElement("h2").appendChild(
+      document.createTextNode("Comments"));
+
+    commentsOutput.appendChild(cmtIntroDOM.parentElement);
+
+    var cmtDOM;
```

Comment:
It looks like you can remove this and use `const` to declare this when it's assigned a few lines down.